src/docs/product/relay/options.mdx

```diff
@@ -6,34 +6,42 @@ redirect_from:
-| relay.mode                 | String  | default value: `managed` possible values: `managed, static, proxy, capture` optional: `false` | Controls how Relay obtains the project configuration for events. For detailed explanation of these modes, see [Relay Modes](/product/relay/modes/).                                                                                                                                                                                                                                                                                             |
-| relay.upstream             | String  | default value: `https://sentry.io` optional: `false`                                          | Fully qualified URL of the upstream Relay or Sentry instance. <Alert level="warning" title="Important"> Relay doesn't check for cycles. Don't set this option to an endpoint that will cause events to be cycled back here. </Alert>                                                                                                                                                                                                            |
-| relay.host                 | String  | default value: `0.0.0.0  in Docker, otherwise  127.0.0.1` optional: `false`                   | The host to which Relay should bind (network interface).                                                                                                                                                                                                                                                                                                                                                                                        |
-| relay.port                 | Integer | default value: `3000` optional: `false`                                                       | The port to which Relay should bind for the unencrypted Relay HTTP server.                                                                                                                                                                                                                                                                                                                                                                      |
+| relay.mode                 | String  | default value: `managed` possible values: `managed, static, proxy, capture` optional: `false` | Controls how Relay obtains the project configuration for events. For detailed explanation of these modes, see [Relay Modes](/product/relay/modes/). `Environment variable: "RELAY_MODE."`                                                                                                                                                                                                                                                       |
```

Comment:
> optional: `false`

Seems confusing to me. What about `optional: no`? Typing the backticks suggests you can set `false`.